app/views/users/_user.json.jbuilder

```diff
@@ -0,0 +1,2 @@
+json.extract! user, :id, :name, :email, :phone, :spire, :active, :admin, :created_at, :updated_at
```

Comment:
Here too with the `jbuilder`, if you don't have immediate plans to use it, it's trivial to add in later. I would just remove it now.